### ISSUE TITLE
Escaped characters in HTML report

### ISSUE THREAD

**Issue by clapierre** [daisy/ace#161]
RC1

serious | EPUB | metadata-schema-accessibilitysummaryAce | "undefined"OEBPS/package.opf
-- | -- | -- | -- | Add a &#39;schema:accessibilitySummary&#39; metadata property to the Package Document

**Comment by clapierre** [comment]
Also what is "undefined" in Location mean?

**Comment by rdeltour** [comment]
@clapierre, I'm not sure I understand the issue. Paste error maybe? Can you please elaborate or send an input file?

Also, "undefined" in the location column means that the document’s title is missing.

**Comment by clapierre** [comment]
aww sorry the GitHub comment converted it correctly
this is what we see in the HTML Report (adding extra space so it shows here

Add a & #39;schema:accessibilitySummary& #39; metadata property to the Package Document

**Comment by rdeltour** [comment]
@marisademeglio can you look into that? I suspect it is in the HTML report generation…

**Comment by marisademeglio** [comment]
@clapierre @rdeltour yes there was a problem with those characters appearing so I fixed that recently, but maybe the fix causing new issues. I'm guessing you haven't seen this behavior before in previous releases? Do you think you could send me an input file?

**Comment by clapierre** [comment]
Should be very easy to reproduce as all EPUBs currently do not have an accessibilitySummary right?
If it is some weird combination that may only appear with a certain combo of errors/warnings I am happy to share the file.  Let me know if it is needed.

**Comment by clapierre** [comment]
I didn't notice this before, but this error also occurred in 0.9.0, however in 0.3 it behaved correctly.
Sorry I don't have anything in-between.

**Comment by rdeltour** [comment]
the quotes render fine for me, tested on Safari 11.0.3, Firefox 58.0 and Chrome 63.0.

Can you please share your OS+browser versions, and maybe send a sample report? (to see if the report’s content is indeed different from what I get).

**Comment by clapierre** [comment]
Ok, so it seems to happen intermittently in Safari and Chrome.  I haven't been able to reproduce the issue in FF.
In Safari I see the escape characters 90% of the time then it works correctly on a reload, but not always.

I guess chalk this up to browser and timing issues.  there are over 102 pages of errors and warnings for this EPUB which may be part of the reason why this is occasionally happening.

**Comment by clapierre** [comment]
BTW Safari was 11.0.2, and Chrome is  63.0.3239.132 (Official Build) (64-bit), FF 58.0

**Comment by rdeltour** [comment]
This is apparently an edge case due to the large file of the report.

I'm moving this out of the 1.0 milestone until we can further investigate. We’ll try to fix that in a dot release.

Charles, can you send us the report file?

**Comment by marisademeglio** [comment]
I can reproduce it consistently if I open the report and then apply filters to the table using the dropdowns. The escape characters then show up, whereas they didn't initially. 

However, I am not sure why this is happening. The initial vs filtered table views are populated from the same source data. The data is initially escape()'d because otherwise the whole webpage halts when it reaches a description about a "Missing < title > element", and so escaping takes care of the angle brackets. I've tried both unescaping the data and not unescaping the data, neither of which seems to have an effect on the behavior reported here.
